bip-cc.md

```diff
@@ -0,0 +1,207 @@
+output Script that lets anyone spend the funds, or burns them.  They have also been non-standard for
+6 years at the time this is written. It was suggested the known vulnerabilities could instead be
+mitigated by committing to the Merkle tree depth in the header's version field[^9]. The authors
+believe it is preferable to address the root cause by invalidating 64 bytes transactions. This
+approach also fixes the vulnerability without developers of SPV verifiers having to implement the
+mitigation or to know it is necessary in the first place.
+
+Several blocks prior to [bip-0034][BIP34] activation contain a coinbase transaction such as its
+scriptSig contains a valid [bip-0034][BIP34] commitment to a future block height. This offers an
+opportunity to duplicate these coinbase transactions in the future[^10] and for this reason
+[bip-0030][BIP30] validation will need to be re-activated from block 1,983,702. A simple way to
+prevent this is to mandate that future coinbase transactions vary slightly from coinbase
+transactions before [bip-0034][BIP34] activation. There are multiple ways of achieving this, but
+enforcing the timelock for the coinbase transaction makes it so all coinbase transactions past
+Consensus Cleanup activation could not have been valid before this height and therefore cannot be a
+duplicate[^11].
+
+# Backward compatibility
+
+This proposal only tightens the block validation rules: there is no block that is valid under the
+rules proposed in this BIP but not under the existing Bitcoin consensus rules. As a consequence
+these changes are backward-compatible with unupgraded node software. That said, the authors strongly
+encourage node operators to upgrade in order to fully validate all consensus rules.
+
+# Acknowledgements
+
+This document builds upon an [earlier proposal][BIP-XXXX] by one of the authors.
+
+The authors would like to thank everyone involved in researching the most appropriate mitigation for
+each of these bugs. We would like to thank in particular Anthony Towns and Sjors Provoost for their
+direct contributions to this proposal, as well as @0xb10c and Brian Groll for providing the authors
+with data to analyze the proposed mitigations.
+
+# Copyright
+
+This document is licensed under the Creative Commons CC0 1.0 Universal license.
+
+
+[^0]: Block 1,983,702 is the earliest future block which could contain a duplicate coinbase
+transaction while still respecting [bip-0034][BIP34]. See [this post][Delving duplicable] for a list
+of all such future blocks.
+[^1]: Technically this limit *cannot* apply to a coinbase transaction as the size of its sole
+input's scriptSig is limited.
+[^2]: The locktime validation, which is also performed for coinbase transactions, enforces that the
+nLockTime value is the last block at which a transaction is invalid, not the first one at which it
+is valid.
+[^3]: The timewarp attack is described [here][SE timewarp] and the Murch-Zawy attack [here][Delving
```

Comment:
```suggestion
[^3]: The timewarp attack is described [here][SE timewarp] and the Murch–Zawy attack [here][Delving
```

Reply:
Let's discuss those in one place: https://github.com/bitcoin/bips/pull/1800#discussion_r2021743417. Closing this one.